os/mm/mm_heap/mm_heapinfo_utils.c

```diff
@@ -153,6 +153,23 @@ void heapinfo_set_caller_addr(void *address, mmaddress_t caller_retaddr)
 
 }
 
+/****************************************************************************
+ * Name: heapinfo_update_stack_pid
+ *
+ * Description:
+ * when create a stack, update pid of stack
+ * if stack node, pid is negative in heapinfo
+ ****************************************************************************/
+void heapinfo_update_stack_pid(void *stack_ptr, pid_t pid)
```

Comment:
Please add this new API in mm.h header file.

Reply:
Thank you